### PR TITLE
feat: visited page filter in replay

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

We lost the `Current URL` property when getting rid of the old simple filters

## Changes

- Adds a `visited_page` property
- Edits the `valuesEndpoint` to lookup the values of the `$current_url` event
- Expands the `visited_page` filter into a `$pageview` with a `$current_url` property before doing queries on the backend

visited_page

### CODE REVIEW

File: frontend/src/models/propertyDefinitionsModel.ts [hunk 1]
```diff
@@ -361,7 +350,7 @@ export const propertyDefinitionsModel = kea<propertyDefinitionsModelType>([
             }
 
             const propValues: PropValue[] = await api.get(
-                endpoint || getEndpoint(values.currentTeamId, type, propertyKey, eventNames, newInput),
+                constructValuesEndpoint(endpoint, values.currentTeamId, type, propertyKey, eventNames, newInput),
```
Comment (pauldambra) [inline frontend/src/models/propertyDefinitionsModel.ts:353]: slightly different here we now always build the endpoint whereas before we'd accept one if it was present... is that ok?

File: frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx [hunk 2]
```diff
@@ -497,6 +497,11 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                         searchPlaceholder: 'Replay',
                         type: TaxonomicFilterGroupType.Replay,
                         render: ReplayTaxonomicFilters,
+                        valuesEndpoint: (key) => {
+                            if (key === 'visited_page') {
+                                return 'api/event/values/?key=' + encodeURIComponent('$current_url')
```
Comment (pauldambra) [inline frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx:502]: (are we able to|should we) restrict this further to just $pageview events?

**Review by pauldambra — APPROVED** [review]
🦖

File: frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx (hunk 2 above)
Comment (daibhin) [reply frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx:502]: Yep, good call. I think we can add `&event_names=$pageview`

File: frontend/src/models/propertyDefinitionsModel.ts (hunk 1 above)
Comment (daibhin) [reply frontend/src/models/propertyDefinitionsModel.ts:353]: We actually still accept the endpoint if present, that logic has just moved up to https://github.com/PostHog/posthog/pull/23689/files?diff=unified#diff-264ccd2aeb4ad3326204b410669dc1c2305b6339254abefd1dcc35d3eb6e3922R128

Previously if you gave an `endpoint` it wouldn't append the the `value` query param which is the input value when searching. We should always include that when sending the request even if some endpoints ignore it

**Pull request merged**